health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/CHANGELOG.md

```
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this module will be documented in this file.
 
+## 0.3.1 - 2025-2-11
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)